src/pages/documentation/mobile-core/release-notes.md

```diff
@@ -1,5 +1,17 @@
 # Release Notes
 
+## March 31, 2023
+
+### Android Mobile Core 2.1.1
+
+* Fixed crash during Core boot when migrating from v4 Mobile SDK.
+* Update POM with Apache 2 license URL.
+
+### Android Identity 2.0.2
+
+* Fixed issue where Identity does not boot when initial Configuration does not contain a valid `experienceCloud.org` value.
```

Comment:
Fixed an issue where Identity could block the processing of requests when the first loaded configuration had incorrect format.